Resources/doc/installation.rst

```diff
@@ -82,9 +82,10 @@ the MongoDB ODM across your application:
 Authentication
 --------------
 
-If you use authentication on your MongoDB database you can provide username, password, and authentication database in the following way:
+If you use authentication on your MongoDB database you can provide username, 
```

Comment:
Small grammar suggestion: "you can provide the username, password, ..."

I think that will read better.